src/tests/dbus-tests/run_tests.py

```diff
@@ -48,9 +48,9 @@ def setup_vdevs():
     for d in vdevs:
         with open('/sys/block/%s/device/model' %
                     os.path.basename(d)) as model_file:
-            assert model_file.read().strip() == 'storaged_test_d'
+            assert model_file.read().strip() == 'udisks_test_d'
```

Comment:
This should be renamed to "udisks_test_dis". (Model of the targetcli devices is first 15 letters of the disk name.)